iceoryx_posh/include/iceoryx_posh/internal/popo/base_subscriber.inl

```diff
@@ -136,6 +136,12 @@ inline void BaseSubscriber<T, Subscriber, port_t>::releaseQueuedSamples() noexce
     m_port.releaseQueuedChunks();
 }
 
+template <typename T, typename Subscriber, typename port_t>
+void BaseSubscriber<T, Subscriber, port_t>::releaseChunk(const mepoo::ChunkHeader* header) noexcept
+{
+    m_port.releaseChunk(header);
+}
```

Comment:
Is this really from you? This looks suspiciously like something from a recently merged PR. Might be an oddment from a merge party.